apps/web/lib/emails/templates/organizer-scheduled-email.ts

```diff
@@ -57,7 +57,7 @@ export default class OrganizerScheduledEmail {
     // Taking care of recurrence rule beforehand
     let recurrenceRule: string | undefined = undefined;
     if (this.recurringEvent?.count) {
-      recurrenceRule = new rrule(this.recurringEvent).toString();
+      recurrenceRule = new rrule(this.recurringEvent).toString().replace("RRULE:", "");
```

Comment:
RRule.js `toString` function returns the recurrence rule with the "RRULE:" prefix and the `createEvent`from `ics.js` appends that too. Removing it to fix it.